_data/simple-icons.json

```diff
@@ -1898,7 +1899,7 @@
         {
             "title": "Conda-Forge",
             "hex": "000000",
-            "source": "https://github.com/conda-forge/conda-forge.github.io/"
+            "source": "https://github.com/conda-forge/conda-forge.github.io/blob/master/img/anvil_black.png"
```

Comment:
I would prefer the SVG version for this one. Also, for Github links, it is better to use the link with commit history part of the url:
https://github.com/conda-forge/conda-forge.github.io/blob/34adb68298ca266af13c3d615f7af8b6c232f6fb/img/anvil.svg

Reply:
The Github preview is not displaying the icon correctly since it is out of box, but the icon is exactly the same we have:
![image](https://user-images.githubusercontent.com/12817388/131843980-0859f95e-ff96-41d3-b6e5-c5403ca7f2a6.png)

Reply:
in that case we can change the source.
Why would the svg be out of the view box?

Reply:
Please wait before resolving conversations.
It is probably only a mistake from the creator which position the icon weirdly in the viewbox. It is still better than the PNG.